### ISSUE TITLE
Does not compile feature in pt-BR when the word contains accents

### ISSUE THREAD

**Issue by tulavalle** [reqnroll/Reqnroll#23]
### Reqnroll Version

1.0.0

### Which test runner are you using?

xUnit

### Test Runner Version Number

2.5.6

### .NET Implementation

.NET 8.0

### Test Execution Method

Visual Studio Test Explorer

### Content of reqnroll.json configuration file

{
  "$schema": "https://schemas.reqnroll.net/reqnroll-config-latest.json",

  "language": {
    "feature": "pt-BR"
  }
}

### Issue Description

Does not compile feature in pt-BR when the word contains accents:

Funcionalidade: Calculator

Simple calculator for adding two numbers

@mytag
Cenário: Add two numbers
	Dado the first number is 50
	E the second number is 70
	Quando the two numbers are added
	Então the result should be 120

### Steps to Reproduce

Write a feature using gherkin reserved words in the pt-BR language:

compile the project
and check if an error occurs:

![image](https://github.com/reqnroll/Reqnroll/assets/108248997/4f769e64-c13c-496a-91c7-b0fba1ebb4f8)

However, there is no error if I remove accents, but it is a spelling error in this language:

![image](https://github.com/reqnroll/Reqnroll/assets/108248997/51f90a78-cd8f-4ad1-97d3-adcdd56a4b01)

### Link to Repro Project

https://github.com/tulavalle/ReqnrollSelenium.git

**Comment by gasparnagy** [comment]
@tulavalle This is strange, because the parsing of the files is not done by Reqnroll itself, but by the Cucumber Gherkin parser. Was this working with SpecFlow before or is this a new project?

**Comment by tulavalle** [comment]
Hi @gasparnagy ! It was a new project, I ran tests here and restarted the machine, it seems to have returned to normal: 
![image](https://github.com/reqnroll/Reqnroll/assets/108248997/b08938c9-d31a-49df-80fe-fe39b641a196)
 It even normalized the developer console, after installing the Reqnroll extension it was no longer opening. I will create a new project in a few days and see if the same thing happens. Any eventuality I will share here!

Note: The same did not happen when migrating projects, I migrated large projects and it was very easy and uneventful. I needed to revert to Specflow because I depend on LivingDoc to publish the documentation for the LivingDoc extension on Azure. As soon as this functionality is implemented I will definitely migrate, congratulations on the work with Reqnroll, Specflow is very good and ReqnRoll is even better!
